README.rst

```diff
@@ -1,130 +1,165 @@
-You might also want to visit `our website <http://coala-analyzer.org/>`_.
+.. image:: https://cloud.githubusercontent.com/assets/7521600/15992701/ef245fd4-30ef-11e6-992d-275c5ca7c3a0.jpg
+
+coala: Language Independent Code Analysis
+-------------
+**coala provides a common command-line interface for linting and fixing all
```

Comment:
@sils1297 This is from the engagement thing. IMO `provides an unified command-line...` sounds better.